### ISSUE TITLE
How do I actually compile a ROM, not a patch?

### ISSUE THREAD

**Issue by Videogamer555** [RPGHacker/asar#286]
When I type asar.exe (with no filename), instead of saying something like "error no asm code file found", or even displaying a list of available commandline switches, it instead goes into some weird IPS patcher mode. It seems from this alone that this software (despite being called an assembler) is really more of a patching tool. I'm looking for a good quality assembler to write my own ROMs, not to patch existing ROMs. Is it possible to do that with asar? If not, do you have any alternative to recommend?

**Comment by Alcaro** [comment]
Yes, Asar is primarily designed for patching, but it works fine for homebrews too. Simply use a nonexistent file as output.
